### ISSUE TITLE
XFCE corrections for TraditionalOk

### ISSUE THREAD

**Issue by chigle** [mate-desktop/mate-themes#152]
Please see below some suggestions / workarounds that fix a few bugs and contrast issues in Xfce 4.12 (Xubuntu 16.04; GTK 3.18 & 2.24) with TraditionalOk Theme. Hope it is helpful.

1. Corrections for GTK-2.0 (gtk.rc)
```
##########################################################################
# Plain White Desktop Labels (inspired by the standard Greybird theme)
##########################################################################
# Xfce specific theming
gtk-color-scheme = "label_fg_color:#000000\nlabel_shadow_color:#ffffff"
style "xfdesktop-icon-view" {
    XfdesktopIconView::label-alpha = 0
    XfdesktopIconView::selected-label-alpha = 80
    XfdesktopIconView::shadow-x-offset = 0
    XfdesktopIconView::shadow-y-offset = 1
    XfdesktopIconView::selected-shadow-x-offset = 0
    XfdesktopIconView::selected-shadow-y-offset = 1
    XfdesktopIconView::shadow-color = shade(1.5, @label_fg_color)
    XfdesktopIconView::selected-shadow-color = shade(1.8, @label_fg_color)
    XfdesktopIconView::shadow-blur-radius = 2
    XfdesktopIconView::cell-spacing = 2
    XfdesktopIconView::cell-padding = 6
    XfdesktopIconView::cell-text-width-proportion = 1.9
    fg[NORMAL] = shade (0.9, @label_shadow_color)
    fg[ACTIVE] = @label_shadow_color
}
widget_class "*XfdesktopIconView*" style "xfdesktop-icon-view"

##########################################################################
# Correct Main Menu Button Size on the Panel (see also Greybird theme)
##########################################################################
style "xfce-panel"  {
    xthickness = 1
    ythickness = 1
}
# XFCE panel theming
widget "*Xfce*Panel*" style "xfce-panel"
class "*Xfce*Panel*" style "xfce-panel"
```

2. Corrections for GTK-3.0 (gtk-widgets.css)
```
/*===========================================================================
Avoid Poor-contrast White Text in Tabs in LibreOffice 5.1/5.2
============================================================================*/
.notebook tab GtkLabel {
    color: @theme_text_color;
}
/*===========================================================================*/


```

**Comment by raveit65** [comment]
Nice, i'm not using xfce and I'm happy about corrections for the old code which isn't from me.
Does the fix for LO works with gtk+-3.20/22 ?
I expect it should be 'notebook tab label {}'  to match the new syntax with 3.20.

**Comment by raveit65** [comment]
No answer from reporter!
